client/src/main/java/org/apache/rocketmq/client/impl/consumer/ConsumeMessagePopConcurrentlyService.java

```diff
@@ -457,6 +457,12 @@ public void run() {
                 consumeMessageContext.getProps().put(MixAll.CONSUME_CONTEXT_TYPE, returnType.name());
             }
 
+            if (ConsumeMessagePopConcurrentlyService.this.defaultMQPushConsumerImpl.hasHook()) {
```

Comment:
The condition is the same to previous branch, it would be better to combine them.

Reply:
@caigy  OK, I will combine them.